notebooks/1-lightcurves/TIKEBook_Lesson1.ipynb

```diff
@@ -0,0 +1,1265 @@
+{
```

Comment:
<p>Potentially a better homework problem here would be the exercise I mention above. Get the cutout from TESSCut for a known transiting e.g., hot Jupiter, create an aperture by-hand to calculate only the flux from the star of interest (rather than including all pixels), and see if you can get a transit.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/16/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>

Reply:
<p>We dont want them copying and pasting code from the notebook to do the homework questions. Also this question was written specifically for this notebook, I would have to ask if it can be changed. </p>